src/test/java/org/sagebionetworks/bridge/exporter/helper/BridgeHelperTest.java

```diff
@@ -64,6 +71,9 @@ private static BridgeHelper setupBridgeHelperWithSession(Session session) {
         return bridgeHelper;
     }
 
+    ArgumentCaptor<RecordExportStatusRequest> requestArgumentCaptor = ArgumentCaptor.forClass(RecordExportStatusRequest.class);
```

Comment:
It's not clear to me that TestNG creates a new test class instance before each test. You should initialize this in a @BeforeMethod instead.